### PR TITLE
Update update-payment.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated roles needed for following the steps

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6279

File: microsoft-365/business-video/update-payment.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ description: "Learn how to update your payment method for Microsoft 365 for busi
 
 You can easily update the payment method for your Microsoft 365 Business subscription. You can change details such as the credit card used, the name, or the address.
 
+## Before you begin
+
+You must be a Global or Billing admin to do the tasks in this article. For more information, see [About admin roles](../../admin/add-users/about-admin-roles.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/update-payment.md]:
```suggestion
You must be a Global or Billing admin to do the tasks in this article. For more information, see [About admin roles](/microsoft-365/admin/add-users/about-admin-roles).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**